src/plugins/securestorage.ts

```diff
@@ -0,0 +1,94 @@
+import { CordovaInstance, Plugin } from './plugin';
+
+declare var cordova: any;
+
+/**
+ * @name Secure Storage
+ * @description
+ * This plugin gets, sets and removes key,value pairs from a device's secure storage.
+ *
+ * Requires Cordova plugin: `cordova-plugin-secure-storage`. For more info, please see the [Cordova Secure Storage docs](https://github.com/Crypho/cordova-plugin-secure-storage).
+ *
+ * @usage
+ * 
+ * ```typescript
+ * import { SecureStorage } from 'ionic-native';
+ *
+ * let secureStorage: SecureStorage = new SecureStorage();
+ * secureStorage.create('my_store_name')
+ *  .then(
+ *    () => console.log('Success'), 
+ *    error => console.log(error);
+ * );
+ * 
+ * secureStorage.get('myitem')
+ *  .then(
+ *    data => console.log(data),
+ *    error => console.log(error)
+ * );
+ * 
+ * secureStorage.set('myitem', 'myvalue')
+ *  .then(
+ *    data => console.log( data),
+ *    error => console.log(error)
+ * );
+ * 
+ * secureStorage.remove('myitem')
+ * .then(
+ *    data => console.log(data),
+ *    error => console.log(error)
+ * );
+ * ```
+ */
+@Plugin({
+  plugin: 'cordova-plugin-secure-storage',
+  pluginRef: 'plugins.securestorage',
+  repo: 'https://github.com/Crypho/cordova-plugin-secure-storage',
+  platforms: ['Android', 'iOS', 'Windows Phone', 'Browser']
+})
+export class SecureStorage {
+
+  // Properties
+  private _objectInstance: any;
```

Comment:
@marcinwadon This is how I designed the `CordovaInstance` decorator to work. To change this we would have to modify the other plugins. We can keep this for now and maybe modify all the plugins later.